blob/service_test.go

```diff
@@ -492,6 +496,66 @@ func TestService_GetAllWithoutPadding(t *testing.T) {
 	}
 }
 
+// TestRetrieveBlobsFromRealEds takes the real data, submitted to mocha-4 and
+// ensures that all blobs under `0x51cc1a73d65f95f6bde6` namespace will be retrieved successfully.
+// At the first step it retrieves all blobs under the given namespace:
+// the amount of the blobs is known and equal to 6. Then it ensures that each blob has a correct index inside the eds
+// by requesting share and comparing them.
+func TestRetrieveBlobsFromRealEds(t *testing.T) {
+	f, err := os.OpenFile("example/eds.json", os.O_RDONLY, os.ModePerm)
```

Comment:
The rule of thumb is to use embed directive for tests. It allows to make all the tests compiled in the binary instead of reading this in runtime.

Reply:
removed this test with json at in four of generated eds with multiple padding shares